README.md

```diff
@@ -1,2 +1,71 @@
-# .github
-Organization-wide repository for common governance documents.
+# Notary Project Overview
+
+The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains by using authentic container images and other OCI artifacts. Notation Project specification and tooling provides signing and verification workflows for OCI artifacts, signature portability across OCI compliant registries, and integration with 3rd party key management solutions through a plugin model.
+
+The Notary Project started with an implementation for signing images in container registries and ensuring their integrity before deployment. The initial implementation uses [The Update Framework (TUF)](https://github.com/theupdateframework) and requires registries to host additional server infrastructure for managing signing keys and TUF metadata. This server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. There is also a client component in the form of the `notary` command line interface (CLI) that can be used by developers or CI/CD pipelines to sign and push container images and update the metadata. The CLI wraps the communication to the registry as well as to the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.
+
+Container images are portable artifacts that can move between registries. Due to the tight integration between the registry, key, and metadata management server component, portability of signatures and therefore images between registries is limited. Container registry APIs have begun standardizing on the [Open Containers Initiative (OCI)](https://github.com/opencontainers) and with new capabilities are added. To overcome the portability challenges and enable future flexibility and standardization, the Notary Project community decided to concentrate on specifications for helping enhance the software supply chains and provide reference implementations.
```

Comment:
I suggest we move this definition to the bottom of the page, so we can let readers jump into The notary project  details quickly

Reply:
This is an important requirement that justifies the need for the specification. I would suggest that we keep it here as @gokarnm suggested.